declarations/OpenAI.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "OpenAI",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://openai.com/policies/terms-of-use",
+      "select": [
```

Comment:
I suggest to use this: 
```json
"select": [
  "#landings-terms-of-use"
],
"remove": [
  "header",
  "[aria-hidden=true]",
  "footer"
],
```

To also include the main title and the updated date

Reply:
Absolutely right 👍